packages/fxa-content-server/app/scripts/lib/metrics.js

```diff
@@ -33,6 +33,13 @@ import xhr from './xhr';
 const SpeedTrap = function() {};
 SpeedTrap.prototype = speedTrap;
 
+// Some integrations, such as Fx for iOS, close the WebView
+// as soon as a login notification is sent to the browser, preventing
+// events from being properly flushed. Instead of waiting for an
+// unload or for an activity timeout, flush every time one of these
+// events comes in.
+const IMMEDIATE_FLUSH_EVENTS = /^(?:(?:screen|flow)\..*|.*\.(?:complete|success))$/;
```

Comment:
With local testing, submitting on every flow event makes this *really* chatty. I wonder if I should only go with screen transitions and complete/success events.